tests/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "playwrightframework",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@playwright/test": "^1.26.0",
+    "eth-testing": "^1.5.0",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0"
+  },
+  "scripts": {
+    "postinstall": "yarn bundle",
+    "bundle": "npx webpack --config webpack.config.js",
+    "test": "yarn bundle && npx playwright test --project=chromium --headed",
+    "test:runner": "yarn bundle && FORCE_COLOR=0 playwright test --project=chromium",
+    "show-results": "npx playwright show-results"
+  },
+  "dependencies": {
+    "@ethersproject/experimental": "^5.7.0",
+    "@ethersproject/providers": "^5.7.2",
+    "@ethersproject/wallet": "^5.7.0",
+    "axios": "^1.2.2",
+    "dotenv": "^16.0.3",
+    "dotenv-webpack": "^8.0.1",
+    "ethers": "^5.7.2",
```

Comment:
@lipmaneth we will not need and of the `eth*` packages for this simple web2 site testing demo. You can remove them all from this file.